BlocNotes/BlocNotes/CoreDataStack.m

```diff
@@ -0,0 +1,104 @@
+//
+//  CoreDataStack.m
+//  BlocNotes
+//
+//  Created by Eddy Chan on 6/26/16.
+//  Copyright © 2016 chan-e. All rights reserved.
+//
+
+#import "CoreDataStack.h"
+
+@implementation CoreDataStack
+
+#pragma mark - Core Data stack
+
+@synthesize managedObjectContext = _managedObjectContext;
+@synthesize managedObjectModel = _managedObjectModel;
+@synthesize persistentStoreCoordinator = _persistentStoreCoordinator;
+
++ (instancetype)sharedStack {
```

Comment:
Great idea starting with this out of the box. You will need it later for the share extension anyway!